includes/admin/class-rop-admin.php

```diff
@@ -1003,6 +1005,76 @@ public function rop_dismiss_cron_disabled_notice() {
 
 	}
 
+	/**
+	 * Migrate the taxonomies from General Settings to Post Format for Pro users.
+	 * @since 8.5.4
+	 */
+	public function migrate_taxonomies_to_post_format() {
+
```

Comment:
Since this is essentially a migration, we can also check if the current installed_at version in the DB is less than the version we'll be releasing this in (8.5.4) and then return if it is

Reply:
Ok, I will add the check.